internal/query/authn_key.go

```diff
@@ -169,8 +169,8 @@ func (q *Queries) searchAuthNKeys(ctx context.Context, queries *AuthNKeySearchQu
 	case JoinFilterUnspecified:
 		// Select all authN keys
 	case JoinFilterApp:
-		joinCol := ProjectColumnID
-		query = query.Join(joinCol.table.identifier() + " ON " + AuthNKeyColumnIdentifier.identifier() + " = " + joinCol.identifier())
+		joinCol := AppColumnID
+		query = query.Join(joinCol.table.identifier() + " ON " + AuthNKeyColumnObjectID.identifier() + " = " + joinCol.identifier())
```

Comment:
looks like the join also did not include the `instance_id`, which can be done using our `join` helper function:
```suggestion
		query = query.Join(join(AppColumnID, AuthNKeyColumnObjectID))
```
please also update the machine case below